code/modules/mob/living/carbon/superior_animal/attack.dm

```diff
@@ -58,9 +58,9 @@
 		else
 			return
 
-/mob/living/carbon/superior_animal/proc/OpenFire(firing_target)
+/mob/living/carbon/superior_animal/proc/OpenFire(var/atom/firing_target)
```

Comment:
```suggestion
/mob/living/carbon/superior_animal/proc/OpenFire(atom/firing_target)
```
?

Reply:
thats just what i do, it works

Reply:
accully why is it an atom?

Reply:
Atoms include turfs, objects and mobs.

Basically it supports more kinds of targets than just mobs.